access/src/main/java/org/apache/accumulo/access/Authorizations.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.accumulo.access;
+
+import java.util.Collection;
+import java.util.Set;
+
+/**
+ *
+ * @since ????
+ */
+public class Authorizations {
+  private final Set<String> authorizations;
+
+  private Authorizations(Set<String> authorizations) {
+    this.authorizations = authorizations;
```

Comment:
Should this be immutable?

Reply:
Yeah it should.  The builder methods where passing in immutable sets. I added Set.copyOf() in 93fd093 which at this point will be a no-op because the builders pass in sets created using Set.of().   It documents the expectation and makes the code more robust against future changes.